<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwhContrast</class>
 <widget class="QWidget" name="PwhContrast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>产品信息结构树：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="tree_product"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_add_master">
         <property name="text">
          <string>N/A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add">
         <property name="text">
          <string>添加对比</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_2">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_add_slave1">
         <property name="text">
          <string>N/A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_2">
         <property name="text">
          <string>添加对比</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_3">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_add_slave2">
         <property name="text">
          <string>N/A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_3">
         <property name="text">
          <string>添加对比</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_4">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>概览</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item>
        <property name="text">
         <string>总工时</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>增值/非增值</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item>
        <property name="text">
         <string>增值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>非增值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>操作类别</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </item>
       <item>
        <property name="text">
         <string>加工</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>检查</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>搬运</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>等待</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="table_master"/>
     </item>
     <item row="1" column="2">
      <widget class="QTableWidget" name="table_slave1"/>
     </item>
     <item row="1" column="3">
      <widget class="QTableWidget" name="table_slave2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
